android/src/main/java/com/imagepicker/Utils.java

```diff
@@ -338,16 +343,45 @@ public static boolean isCameraPermissionFulfilled(Context context, Activity acti
     }
 
     static boolean isImageType(Uri uri, Context context) {
+        String imageMimeType = "image/";
+
+        if (uri.getScheme().equals("file")) {
```

Comment:
added because this function is also called from `getResponseMap` when creating an image/ video. in that case, the content resolver doesnt work